### PR TITLE
Add helper to safely retrieve post or URL query fields if set | #68051

### PR DESCRIPTION
**Pull request by barryhughes**
Adds a helper to retrieve post fields/URL query fields if set.

https://central.tri.be/issues/68051

### CODE REVIEW

File: src/functions/utils.php [hunk 1]
```diff
@@ -123,3 +123,27 @@ function tribe_exit( $status = '' ) {
...
+	 * Post data takes priority over fields passed in the URL query. If the field is not
+	 * set then $default (null unless a different value is specified) will be returned.
+	 *
+	 * @param string $var
+	 * @param mixed  $default
+	 *
+	 * @return mixed
+	 */
+	function tribe_get_request_var( $var, $default = null ) {
+		if ( isset( $_POST[ $var ] ) ) {
```
Comment (bordoni) [inline src/functions/utils.php]: Could you use the `Utils__Array::get()` from: https://github.com/moderntribe/tribe-common/blob/release/4.3.3/src/Tribe/Utils/Array.php#L70

It's on our current MR, allows for deep search on an Array.

Comment (barryhughes) [inline src/functions/utils.php]: ...to expand on that, if we have `type[fruit]=apple&type[vegetable]=pear` I don't necessarily want to get back `apple` if I pass in `fruit` as the var name.

Comment (barryhughes) [inline src/functions/utils.php]: Actually, realizing I totally misunderstood how Utils__Array worked: yep we could use that, but it's not it develop yet (I can add a note to update once it is, though).

Comment (bordoni) [inline src/functions/utils.php]: It will be tonight, would you mind waiting to merge this after that is updated?

**Comment by bordoni** [comment]
@barryhughes  just merged `release/4.3.3` into `develop` now we can use `Utils__Array`

**Comment by borkweb** [comment]
:+1:

**Pull request merged**